runtime/src/main/resources/application.properties

```diff
@@ -3,6 +3,8 @@
 # SPDX-FileCopyrightText: Huawei Inc.
 #
 #
+
+spring.profiles.active=zitadel
```

Comment:
it must not be activated by default, It must only be activated on demand.

Reply:
OK.